templates/tags-input.html

```diff
@@ -1,9 +1,9 @@
-<div class="host" tabindex="-1" ng-click="eventHandlers.host.click()" ti-transclude-append>
+<div class="host" tabindex="-1" ng-click="disabled || eventHandlers.host.click()" ti-transclude-append ng-class="{'disabled': disabled}">
   <div class="tags" ng-class="{focused: hasFocus}">
     <ul class="tag-list">
       <li class="tag-item" ng-repeat="tag in tagList.items track by track(tag)" ng-class="{ selected: tag == tagList.selected }">
         <span ng-bind="getDisplayText(tag)"></span>
-        <a class="remove-button" ng-click="tagList.remove($index)" ng-bind="options.removeTagSymbol"></a>
+        <a class="remove-button" ng-click="tagList.remove($index)" ng-bind="options.removeTagSymbol" ng-hide="disabled"></a>
```

Comment:
I'm not sure whether hiding the remove buttons is the best approach. I imagine that showing/hiding them will mess with the element layout, since tags will be "shorter" without the button next to them.